tests/Core/Components/Slider/FluentSliderTests.razor

```diff
@@ -0,0 +1,212 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using Xunit;
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Samples;
+@inherits TestContext
+@typeparam TValue
+@code
+{
+	public FluentSliderTests()
+	{
+		JSInterop.Mode = JSRuntimeMode.Loose;
+		Services.AddFluentUIComponents();
+	}
+
+	[Fact]
+	public void FluentSlider_Default()
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentSlider Value=10>fluent-Slider</FluentSlider>);
+
+		// Assert
+		cut.Verify();
+	}
+
+	[Fact]
+	public void FluentSlider_Min()
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentSlider Min="2" Value=10 />);
+
+		// Assert
+		cut.Verify();
+	}
+
+	[Theory]
+	[InlineData(SliderSize.Medium)]
+	[InlineData(SliderSize.Small)]
+	[InlineData((SliderSize)999)]
+	public void FluentSlider_Size(SliderSize sliderSize)
+	{
+		// Arrange && Act
+		using var cut = Render(@<FluentSlider Value=10 Size="@sliderSize" />);
+
+		// Assert
+		cut.Verify(suffix: sliderSize.ToString());
+	}
+
+	[Theory]
+	[InlineData(true)]
+	[InlineData(false)]
+	public void FluentSlider_DisabledAttribute(bool disabled)
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentSlider Value=10 Disabled="@disabled" />);
+
+		// Assert
+		var actual = cut.Find("fluent-slider").HasAttribute("disabled");
+		Assert.Equal(disabled, actual);
```

Comment:
Can you remove the unused ".verified.html" files (here :  `...disabled...` file)?